src/ziggurat/streams.clj

```diff
@@ -244,7 +245,7 @@
                    (.setUncaughtExceptionHandler stream
                                                  (reify StreamsUncaughtExceptionHandler
                                                    (^StreamsUncaughtExceptionHandler$StreamThreadExceptionResponse handle [_ ^Throwable error] (handle-uncaught-exception (get stream-config :stream-thread-exception-response :shutdown-client) error))))
-                   (.start stream)
+                   (clog/with-logging-context {:consumer-group topic-entity} (.start stream))
```

Comment:
This should also be in mapper.clj - where the actual call to the handler is made.

Similarly we can add `with-context` to rabbitmq publish and consum flow as well.

Reply:
can't this be just added at one place like in mapper or stream clj and then all the logs automatically add this to their log line? Right now it is duplicated at so many places.